script/Puzzledata.js

```diff
@@ -0,0 +1,41 @@
+const { db, models: {Puzzle}} = require('../server/db');
+const { faker } = require('@faker-js/faker');
+
+//fake Puzzle data
+let puzzles = [];
+for (let i = 0; i < 50; i ++) {
+    let newPuzzle = {
+        name: faker.lorem.word({length: { min: 2, max: 5}}),
+        description: faker.lorem.paragraph(),
+        stockQuantity: faker.datatype.number({min: 0}),
+        puzzlePieces: faker.helpers.arrayElement(['250 pieces','500 pieces', '1000 pieces']),
+        price: faker.helpers.arrayElement(['$12.99', '$15.99', '$19.99']),
+          //if puzzlePieces = 250 pieces, return $12.99  
+          //if puzzlePieces = 500 pieces, return $15.99 
+          //if puzzlePieces = 1000 pieces, return $19.99 
+        imgURL: 'https://picsum.photos/200',
```

Comment:
I think because you're setting it as a defaultValue we probably don't need to specify it here :)  - same with the Puzzle.create one in line 31